synapse/lib/storm.py

```diff
@@ -896,11 +894,9 @@ def _cmprCtorIvalIval(self, oper):
         def cmpr(tufo):
 
             minv = tufo[1].get(minp)
-            if minv is None:
-                return False
-
             maxv = tufo[1].get(maxp)
-            if maxv is None:
+
+            if minv is None or maxv is None:
```

Comment:
A bit of the above checking ( only checking one for None ) was a purposeful speed optimization ( because an interval *must* have both ) but i'm not against the explicit checks.  Small perf hit for explicit clarity.